pkg/cmd/roachtest/roachtestutil/mixedversion/planner.go

```diff
@@ -192,11 +192,20 @@ const (
 	LastUpgradeStage
 	RunningUpgradeMigrationsStage
 	AfterUpgradeFinalizedStage
+	UpgradingSystemStage
```

Comment:
Do these two upgrading stages still adhere to `Upgrade stages are defined in the order they happen during test runs`? 

From my understanding, if a tenant is on `UpgradingSystemStage`, then it hasn't yet started any upgrades so this stage should come earlier. But I could also see why making comparisons against these two stages don't make sense.

Reply:
Good catch. As you said, it doesn't really make sense to perform any kind of comparison with these stages. I gave them an arbitrary large value (so that `<` comparisons are consistently false) and updated the comments to reflect this.

For that matter, I only expect mutators to need to do things that are low level enough to involve comparing stages in this way.